presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetColumnChunk.java

```diff
@@ -65,8 +63,8 @@ protected PageHeader readPageHeader()
     public ParquetColumnChunkPageReader readAllPages()
```

Comment:
I find the name of this class confusing because the name `ParquetColumnChunkPageReader` has both `ColumnChunk` and `Page` in it, which are two separate entities in Parquet. And as far as I understand this only reads pages, so how about renaming this class to `ParquetPageReader` and get rid of the existing `ParquetPageReader` interface, which only has this class as its implementation? Do you envision having other implementations for the existing `ParquetPageReader` interface?